python/src/aiconfig/editor/client/src/components/EditorContainer.tsx

```diff
@@ -562,34 +561,26 @@ export default function EditorContainer({
         }
 
         const promptName = statePrompt.name;
-        const isStream = isStreamingSupported(statePrompt, stateRef.current);
-
-        if (isStream) {
-          await runPromptStreamCallback(promptName, (event) => {
-            if (event.type === "output_chunk") {
-              dispatch({
-                type: "STREAM_OUTPUT_CHUNK",
-                id: promptId,
-                output: event.data,
-              });
-            } else if (event.type === "aiconfig") {
-              dispatch({
-                type: "CONSOLIDATE_AICONFIG",
-                action,
-                config: event.data,
-              });
-            }
-          });
-          return;
-        } else {
-          const serverConfigRes = await runPromptCallback(promptName);
-
-          dispatch({
-            type: "CONSOLIDATE_AICONFIG",
-            action,
-            config: serverConfigRes.aiconfig,
-          });
-        }
+        const enableStreaming = isStreamingSupported(
```

Comment:
Can we change `isStreamingSupported` to something like `getModelSettingsStream` or something similar, and have it return either a boolean or undefined? Thinking here is:
- by default, models that support streaming but don't have 'stream' value set in the settings should still stream in editor to have the best UX by default
- if user explicitly states 'no' for stream value in settings, then we shouldn't stream
- our server implementation can technically support 'stream' style response for all models (so we can handle the undefined 'enableStreaming' param in Editor.tsx `runPrompt` function by defaulting to true. But other external implementations might not, so they can handle the undefined case however they want